server/middlewares/validations.js

```diff
@@ -235,6 +235,29 @@ class inputValidator {
     request.params.id = id;
     return next();
   }
+
+  /**
+   * @description searchValidator
+   * @param {object} request http request object
+   * @param {object} response http response object
+   * @param {object} next callback function
+   * @returns {object} http response object
+   */
+  static searchValidator(request, response, next) {
```

Comment:
shouldnt you validate that author, tag, and keywords are strings?

Reply:
I'm on this now.